core/src/main/java/com/backblaze/b2/client/B2StorageClientImpl.java

```diff
@@ -317,6 +318,11 @@ public B2FileVersion getFileInfo(B2GetFileInfoRequest request) throws B2Exceptio
         return retryer.doRetry("b2_get_file_info", accountAuthCache, () -> webifier.getFileInfo(accountAuthCache.get(), request), retryPolicySupplier.get());
     }
 
+    @Override
+    public B2FileVersion getFileInfoByName(B2GetFileInfoByNameRequest request) throws B2Exception {
+        return retryer.doRetry("b2_get_file_info", accountAuthCache, () -> webifier.getFileInfo(accountAuthCache.get(), request), retryPolicySupplier.get());
```

Comment:
let's say "get_file_info_by_name" here to distinguish it from the other case.
i'm proposing that we leave the "b2_" off the front since it's not really a b2 api call.
i could be convinced otherwise though.  :)

Reply:
done